src/main/java/org/folio/innreach/domain/service/ItemService.java

```diff
@@ -13,4 +13,6 @@
 
   Optional<InventoryItemDTO> findItemByBarcode(String barcode);
 
+  Optional<InventoryItemDTO> find(UUID itemId);
```

Comment:
the method is inherited from `BasicService`, no need to define it

Reply:
It is moved, thank you